pyphare/pyphare/pharesee/particles.py

```diff
@@ -16,17 +16,26 @@ def __init__(self, **kwargs):
             self.v       = np.random.randn(box.size()*100, 3)
             self.weights = np.zeros_like(self.deltas) + 0.01
             self.charges = np.zeros_like(self.weights) + 1
+            self.dl       = np.zeros(box.dim())+0.1
 
         else:
             self.iCells  = kwargs["icells"]
             self.deltas  = kwargs["deltas"]
             self.v       = kwargs["v"]
             self.weights = kwargs["weights"]
             self.charges = kwargs["charges"]
+            self.dl      = kwargs["dl"]
+            self._x = None
```

Comment:
should this be indented?

Reply:
I'm not sure it matters, but I would guess it should be outside the if

Reply:
well I don't know what dl to put to randomely generated particles taht's why I've not put any to them... but we do not use that in tests so for now I think it's ok